course_info/templates/course_info/partials/details.html

```diff
@@ -261,7 +261,6 @@ <h3 class="breadcrumbs">
                       <a href="{{site.course_site_url}}" target="_blank">{{site.course_site_url}}</a>
                     </div>
                     <div class="col-sm-3">
-                      <!-- todo: switch $index to site_id when available -->
                       <button type="button" class="btn btn-default pull-right"
                               ng-click="dc.dissociateSite($index)">
```

Comment:
so, using the index is nice because you get the object in one step 'dc.courseInstance.sites[siteListIndex].site_map_id' doing a lookup seems unnecessary here, I may be missing something though.